src/pages/index.js

```diff
@@ -65,34 +109,137 @@ profileEditBtn.addEventListener("click", () => {
   editModal.open();
 });
 
-//--------------------ADD CARD POPUP-WITH-FORM INSTANTIATION-------------------->>
+//--------------------PROFILE PICTURE POPUP-WITH-FORM INSTANTIATION-------------------->>
+
+const profilePictureModal = new PopupWithForm(
+  "#profile-picture-edit-modal",
+  () => {
+    profilePictureModal.loadingButtonState();
+    api
+      .setProfilePicture(profilePictureFormInput.value)
+      .then((res) => {
+        profileAvatar.src = res.avatar;
```

Comment:
`        profileAvatar.src = res.avatar;
`

Class `UserInfo` should set the entire user’s data (including the **avatar**)